### ISSUE TITLE
Add alt text for img tags

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/training-specific-template#148]
We have lots of screenshots included with `<img>`  but no alt text, and more accessibility >> less accessibility! h/t @jashapiro for thinking of this.

**Comment by sjspielman** [comment]
Closed with #149
